Tools/iobench/iobench.py

```diff
@@ -266,18 +280,17 @@ def run_all_tests(options):
     def print_label(filename, func):
         name = re.split(r'[-.]', filename)[0]
         out.write(
-            ("[%s] %s... "
-                % (name.center(7), func.__doc__.strip())
-            ).ljust(52))
+            f"[{name.center(7)}] {func.__doc__.strip()}... ".ljust(52))
         out.flush()
 
     def print_results(size, n, real, cpu):
         bw = n * float(size) / 1024 ** 2 / real
         bw = ("%4d MiB/s" if bw > 100 else "%.3g MiB/s") % bw
         out.write(bw.rjust(12) + "\n")
         if cpu < 0.90 * real:
-            out.write("   warning: test above used only %d%% CPU, "
-                "result may be flawed!\n" % (100.0 * cpu / real))
+            out.write("   warning: test above used only "
+                      f"{100.0 * cpu / real}% CPU, "
```

Comment:
Little-known fact: Python has a builtin `%` format specifier 

```suggestion
                      f"{cpu / real:%} CPU, "
```

Reply:
Slightly different output but doesn't really matter here:

```pycon
>>> f"{100.0 * 12 / 36}% CPU"
'33.333333333333336% CPU'
>>> f"{12 / 36:%} CPU"
'33.333333% CPU'
```